package.json

```diff
@@ -39,19 +39,19 @@
         "@babel/preset-react": "7.16.7",
         "@babel/register": "^7.16.0",
         "@babel/runtime-corejs3": "7.16.8",
+        "@exabyte-io/ade.js": "2022.11.16-0",
+        "@exabyte-io/code.js": "2022.11.11-0",
+        "@exabyte-io/eslint-config": "^2022.11.17-0",
+        "@exabyte-io/ide.js": "2022.7.28-1",
+        "@exabyte-io/made.js": "2022.6.15-0",
```

Comment:
Do we need to move `ade`, `ide`, `eslint-config`, etc to `dependencies`?
I think `made.js` should be added to `peerDependencies`, but the rest can stay in `devDependencies`

Reply:
I was just fixing with this eslint error for uning import from `made.js`, but not listed in dependencies.
Will do as you suggested.